utils/python/CIME/env_module.py

```diff
@@ -23,7 +21,10 @@ def __init__(self, machine, compiler, cimeroot, caseroot, mpilib, debug=False):
         self._caseroot = caseroot
         self._mpilib   = mpilib
         self._debug    = debug
-
+        self._cshscript = list()
```

Comment:
Add write of .env_mach_specific.csh and .sh - this is needed for tests that are still in csh but it's also a really handy file to have for debugging.